assets/src/edit-story/app/media/utils/useUploadVideoFrame.js

```diff
@@ -54,11 +54,10 @@ function useUploadVideoFrame({ updateMediaElement }) {
         featured_media: posterId,
         post: storyId,
       });
-      if (elementId) {
-        // TODO Update all elements that use video.
-        const newState = { resource: { ...resource, posterId, poster } };
-        setProperties(elementId, newState);
-      }
+      const newState = {
+        resource: { posterId, poster, videoId },
+      };
+      setProperties(videoId, newState);
```

Comment:
Can you pls confirm that the `updateMediaElement` below does the same to resources as the `setProperties` in this line: e.g. finds the video's resource(s) and updates them? I'm asking because I'm confused: this is an old code and I don't understand why you even tried to introduce `resource` argument here initially?